package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
we shouldn't check this file in, not sure how generated?

Reply:
We must check this in, that's how npm works, for better or worse.

Reply:
This is in root dir?

Reply:
Adding that was a mistake, removed.